Jellyfin.Api/Controllers/UserController.cs

```diff
@@ -498,8 +503,14 @@ public async Task<ActionResult<UserDto>> CreateUserByName([FromBody, Required] C
         [ProducesResponseType(StatusCodes.Status200OK)]
         public async Task<ActionResult<ForgotPasswordResult>> ForgotPassword([FromBody, Required] ForgotPasswordDto forgotPasswordRequest)
         {
+            var ip = HttpContext.GetNormalizedRemoteIp();
             var isLocal = HttpContext.IsLocal()
-                          || _networkManager.IsInLocalNetwork(HttpContext.GetNormalizedRemoteIp());
+                          || _networkManager.IsInLocalNetwork(ip);
+
+            if (!isLocal)
+            {
+                _logger.LogWarning("Password reset proccess initiated from outside the local network IP:{ip}", ip);
```

Comment:
```suggestion
                _logger.LogWarning("Password reset proccess initiated from outside the local network IP: {IP}", ip);
```
https://github.com/serilog/serilog/wiki/Writing-Log-Events#message-template-recommendations